### DISCUSSION TITLE
How to copy value to another value?

### DISCUSSION THREAD

**Discussion by sly-fox-git** [beetbox/beets#5124] [Q&A] [answered]
What is the proper command to copy one value to another? For example:

`beet modify album=$title`

**Comment by wisp3rwind** [comment, marked as answer]
This should work, assuming you are running a development version of beets, cf. https://github.com/beetbox/beets/issues/4543. You'll also need to quote the command properly, such that your shell does not expand the variable to something undesirable (like the empty string) already:

```
beet modify album='$title'
```
